README.md

```diff
@@ -22,13 +22,84 @@ mn create-app life.qbic.sampletracking --features=groovy --build maven
 ```
 
 ## Data model
-The data model that holds sample tracking information is denfined by attributes and relations shown in the following ER diagram.
+The data model that holds sample tracking information is defined by attributes and relations shown in the following ER diagram.
 
 ![er-diagram](models/sample-tracking-er.svg)
 
 ## API design
 The remote [RESTful API](https://app.swaggerhub.com/apis-docs/qbic/sample-tracking) is created with [swagger.io](https://swagger.io/).
 
+## Authentication
+
+The roles and user tokens must be provided in a file following the YAML format specification.
+An exemplary entry looks like this:
+
+```
+servicereader:
+    token: 123!
+    roles:
+        - READER
+    servicewriter:
+        token: 123456!
+        roles:
+            - READER
+            - WRITER
+```
+
+## Input Formats
+The accepted input formats are listed in the following
+### For samples
+To obtain tracking information for a given sample id
+```
+{
+"sampleId":"QABCD12AE"
+}
+```
+To set the current location for a sample with the given identifier
+```
+{
+    "sampleId":"QABCD12AE",
+    "location":{
+        "name": "Max Mustermann",
+        "responsiblePerson": "Max Mustermann",
+        "responsibleEmail": "max.mustermann@uni-tubingen.de",
+        "address"{
+            "affiliation": "QBiC",
+            "street": "Auf der Morgenstelle 6",
+            "zipCode": 72076,
+            "country": "Germany"
+        },
+        "status":["SAMPLE_QC_FAIL", "SAMPLE_QC_PASS", "SAMPLE_RECEIVED", "LIBRARY_PREP_FINISHED", "SEQUENCING",
+                    "SEQUENCING_COMPLETE", "METADATA_REGISTERED", "WAITING", "PROCESSING", "FAILED_QC", "PROCESSED",
+                    "DATA_AVAILABLE",],
+        arrivalDate:{
+            "year": 2021,
+            "month": 10, 
+            "date": 5, 
+            "hrs": 15, 
+            "min": 00
+        },
+        forwardDate:{
+            "year": 2021,
+            "month": 10, 
+            "date": 5, 
+            "hrs": 15, 
+            "min": 00
+        }
+    }
```

Comment:
```suggestion
{
  "name":"Example Location Name",
  "responsible_person":"Max Mustermann",
  "responsible_person_email":"max.mustermann@uni-tubingen.de",
  "address":{
    "affiliation":"QBiC",
    "street":"Auf der Morgenstelle 6",
    "zip_code":72076,
    "country":"Germany"
  },
  "sample_status":"WAITING",
  "arrival_date":"2021-12-07T09:38Z",
  "forward_date":"2021-12-07T09:38Z"
}
```
NOTE: The provided location information should be stripped of all newlines("\n") even between the attributes, otherwise it can't be interpreted by the sample-tracking-service

Reply:
The note part should be part of the suggestion, right? Can you update the suggestion?

Reply:
Done good point